test_data/rules/avoid_type_to_string.dart

```diff
@@ -145,6 +145,7 @@ extension ExtensionOnGoodWithTypeAndMixin on GoodWithTypeAndMixin {
 }
 
 extension on int Function(int) {
+  // ignore: unused_element
```

Comment:
You could also name the extension so that it isn't private (unless that's important to the test in some way that I'm not seeing.

Reply:
I wish I knew the intent here.   I'll look a little closer.

(Aside: these are exactly the kinds of questions I hope this work will help avoid in the future!)